### PR TITLE
Added e2e tests for insights-operator

### PR DESCRIPTION
**Pull request by quarckster**
Required for https://github.com/openshift/insights-operator/pull/50

### CODE REVIEW

File: ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ tag_specification:
   namespace: ocp
 tests:
 - as: unit
-  commands: go test ./...
+  commands: make test
```
Comment (mfojtik) [inline ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml]: generally, we use `make test-unit` for unit tests... `make test` normally runs **all** tests

Comment (quarckster) [reply ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml]: Done

**Pull request merged**